Lib/fontTools/ttLib/ttFont.py

```diff
@@ -755,12 +755,6 @@ def draw(self, pen):
 		"""
 		self._glyph.draw(pen)
 
-	def drawPoints(self, pen):
```

Comment:
maybe you could leave this method in for _TTGlyphCFF class and raise NotImplementedError, so instead that an AttributeError the user would get a more meaningful error